<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UploadDialog</class>
 <widget class="QDialog" name="UploadDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>362</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Create new file on drive</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,2,6,1">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_UploadFile">
     <item>
      <widget class="QLabel" name="label_FilePath">
       <property name="text">
        <string>Upload File:</string>
       </property>
       <property name="buddy">
        <cstring>lineEdit_FilePath</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_FilePath"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_FilePath">
       <property name="text">
        <string>Select File</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_UploadParameter">
     <property name="title">
      <string>Upload Parameter</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_uploadType">
        <property name="text">
         <string>uploadType</string>
        </property>
        <property name="buddy">
         <cstring>comboBox_uploadType</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox_uploadType">
        <item>
         <property name="text">
          <string>media(Simple Upload)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>multipart(Multipart Upload)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>resumable(Resumable Upload)</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_OptionalQueryParameters">
     <property name="title">
      <string>Optional query parameters</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_enforceSingleParent">
        <property name="text">
         <string>enforceSingleParent</string>
        </property>
        <property name="buddy">
         <cstring>comboBox_enforceSingleParent</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox_enforceSingleParent">
        <item>
         <property name="text">
          <string>false</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>true</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_ignoreDefaultVisibility">
        <property name="text">
         <string>ignoreDefaultVisibility 	</string>
        </property>
        <property name="buddy">
         <cstring>comboBox_ignoreDefaultVisibility</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBox_ignoreDefaultVisibility">
        <item>
         <property name="text">
          <string>false</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>true</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_keepRevisionForever">
        <property name="text">
         <string>keepRevisionForever</string>
        </property>
        <property name="buddy">
         <cstring>comboBox_keepRevisionForever</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBox_keepRevisionForever">
        <item>
         <property name="text">
          <string>false</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>true</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_ocrLanguage">
        <property name="text">
         <string>ocrLanguage</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_ocrLanguage</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_ocrLanguage"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_supportsAllDrives">
        <property name="text">
         <string>supportsAllDrives</string>
        </property>
        <property name="buddy">
         <cstring>comboBox_supportsAllDrives</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="comboBox_supportsAllDrives">
        <item>
         <property name="text">
          <string>false</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>true</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_useContentAsIndexableText">
        <property name="text">
         <string>useContentAsIndexableText</string>
        </property>
        <property name="buddy">
         <cstring>comboBox_useContentAsIndexableText</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="comboBox_useContentAsIndexableText">
        <item>
         <property name="text">
          <string>false</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>true</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>UploadDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>UploadDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
